plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1310,9 +1329,13 @@ def get_aaa_settings_for_site(self, site_name, site_id):
             network_aaa = aaa_network_response.get("response", {}).get("aaaNetwork")
             client_and_endpoint_aaa = aaa_network_response.get("response", {}).get("aaaClient")
 
-            if not network_aaa or not client_and_endpoint_aaa:
-                self.log("No AAA settings found for site '{0}' (ID: {1})".format(site_name, site_id), "WARNING")
-                return None, None
+            if network_aaa or not client_and_endpoint_aaa:
```

Comment:
if network_aaa is there, then we return without checking the next condition. Is that correct?

Reply:
Yes, it should check both condition. so replaced "or" with "and".